addons/html_editor/static/src/editor/core/delete_plugin.js

```diff
@@ -473,14 +479,25 @@ export class DeletePlugin extends Plugin {
     // Fill empty blocks
     // Remove empty inline elements, unless cursor is inside it
     handleEmptyElements(commonAncestor, cursorPos) {
+        const handleShrunkBlock = (block) => {
+            if (!isShrunkBlock(block)) {
```

Comment:
no big deal, but in this case, i would just do this check at the two locations where it's called

Reply:
Ok, I've changed it.